### ISSUE TITLE
Preconfigured connection is not loaded on first browser page load

### ISSUE THREAD

**Issue by lukashornych** [FgForrest/evitaDB#682]
There is some weird behaviour (at least in Firefox) where on first lab page load, the preconfigured connection is not displayed even though the cookie is sent. There is no error in browser either. However on page reload (not new tab or new address), the connection is displayed correctly.

**Comment by novoj** [comment]
@lukashornych is this an issue for evita core?

**Comment by lukashornych** [comment]
I think so. But it could be combination of evita and lab.

**Comment by novoj** [comment]
Is this issue still valid @lukashornych ?

**Comment by lukashornych** [comment]
@novoj I don't think it is. This was greatly reworked since than and I don't see it anymore. I will close it.
